src/memex/db/__init__.py

```diff
@@ -20,7 +26,14 @@ def init(engine, base=None, should_create=False, should_drop=False):
         base.metadata.create_all(engine)
 
 
-def set_base(basecls):
+def set_base(basecls=None):
```

Comment:
This is exactly why I was bringing up a memex demo app last time, I was thinking that always having to provide a base class might be cumbersome for some more minimal implementations.